app/components/ReportingRequirement/ReportDueIndicator.tsx

```diff
@@ -96,7 +84,7 @@ const ReportDueIndicator: React.FC<Props> = ({
                 {hasValidReportDueDate ? (
                   <>
                     {overdue ? "Overdue by " : "Due in "}
-                    <b>{Math.abs(reportDueIn.days)} day(s)</b>
+                    <b>{Math.abs(reportDueIn)} day(s)</b>
```

Comment:
This will eventually need refactoring in a separate function, so that is displays a number of months/years when appropriate